multigeiger/speaker.cpp

```diff
@@ -84,14 +87,15 @@ void IRAM_ATTR isr_audio() {
     mcpwm_set_frequency(MCPWM_UNIT_0, MCPWM_TIMER_0, frequency_mHz / 1000);
     // start outputting PWM signal(s)
     mcpwm_start(MCPWM_UNIT_0, MCPWM_TIMER_0);
-  } else {
+  } else if (frequency_mHz == 0) {  // speaker off
     // frequency_mHz == 0 -> disable sound output
     // stop any PWM signals
     mcpwm_stop(MCPWM_UNIT_0, MCPWM_TIMER_0);
     // keep A high and B low (we have a piezo, no current flowing)
     mcpwm_set_signal_high(MCPWM_UNIT_0, MCPWM_TIMER_0, MCPWM_OPR_A);
     mcpwm_set_signal_low(MCPWM_UNIT_0, MCPWM_TIMER_0, MCPWM_OPR_B);
   }
+  // frequency_mHz == -1 -> don't touch pwm/speaker
```

Comment:
slight optimization: if user switches off speaker ticks, it won't do any PWM calls here.